openedx/core/djangoapps/catalog/tests/factories.py

```diff
@@ -8,6 +8,7 @@
 
 
 fake = Faker()
+VERIFIED_MODE = 'verified'
```

Comment:
Could you import `CourseMode` and use that instead?

Reply:
I tried that but got import errors when running tests that used the factory

Reply:
@amangano-edx this is the import error I get when I try to import `CourseMode` into `factories.py`

`from common.djangoapps.course_modes.models import CourseMode
E   ImportError: No module named djangoapps.course_modes.models`

Reply:
This is not worth fixing.  Let's move on and merge.  This is a test.

Reply:
`from course_modes.models import CourseMode`

Reply:
^ works for me locally, that's also how it's imported into the test_utils.py file.

Reply:
This seemed to work locally for me as well, but based on the test failures it seems like it is not the solution